tests/compute/test_transform.py

```diff
@@ -574,16 +574,20 @@ def _check(g, new_g, induced_nodes):
 @parametrize_dtype
 def test_to_simple(index_dtype):
     g = dgl.heterograph({
```

Comment:
Let's also add a test for graphs with a single edge type. In particular, writeback mapping should be a tensor in this case.